docs/docs/gatsby-config.md

```diff
@@ -1,26 +1,26 @@
 ---
-title: Gatsby Config API
+title: API Gatsby Link
 ---
 
-Site configuration options for a Gatsby site are placed in a file at the root of the project folder called `gatsby-config.js`.
+Opcje konfiguracji dla strony w Gatsby umieszczone są wewnątrz pliku `gatsby-config.js` umiejscowionego w głównym folderze projektu.
 
-_Note: There are many sample configs which may be helpful to reference in the different [Gatsby Example Websites](https://github.com/gatsbyjs/gatsby/tree/master/examples)._
+_Uwaga: Wewnątrz [Gatsby Example Websites](https://github.com/gatsbyjs/gatsby/tree/master/examples) znajduje się wiele pomocnych przykładowych konfiguracji, do krotóych dla porównania można się odnieść._
```

Comment:
Typo. Change 'krotóych' to 'których'. By the way, sorry but this sentence doesn't sounds good.